bear-language-requirements.yaml

```diff
@@ -0,0 +1,411 @@
+AlexBear:
+  - Natural Language
+AnnotationBear:
+ApertiumLintBear:
+  - Apertium
+ArtisticStyleBear:
+  - C
+  - C#
+  - C++
+  - Java
+  - Objective-C
+BanditBear:
+  - Python
+  - Python 2
+  - Python 3
+BashateBear:
+  - bash
+BootLintBear:
+  - HTML
+CISkipInspectBear:
+  - Git
+CMakeLintBear:
+  - CMake
+CPDBear:
+  - C#
+  - CPP
+  - Fortran
+  - Go
+  - JSP
+  - Java
+  - JavaScript
+  - Matlab
+  - Objective-C
+  - Octave
+  - PHP
+  - PLSQL
+  - Python
+  - Python 2
+  - Python 3
+  - Ruby
+  - Scala
+  - Swift
+CPPCheckBear:
+  - C
+  - C++
+CPPCleanBear:
+  - C++
+CPPLintBear:
+  - C++
+CSSAutoPrefixBear:
+  - CSS
+CSSCombBear:
+  - CSS
+CSSLintBear:
+  - CSS
+CSVLintBear:
+  - CSV
+CSecurityBear:
+  - C
+  - C++
+CSharpLintBear:
+  - C#
+CheckstyleBear:
+  - Java
+ClangASTPrintBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangCloneDetectionBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangComplexityBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+ClangFunctionDifferenceBear:
+  - C
+  - C++
+  - CUDA
+  - Objective-C
+  - Objective-C++
+  - OpenCL
+  - OpenMP
+CoffeeLintBear:
+  - CoffeeScript
+DartLintBear:
+  - Dart
+DennisBear:
+  - po
+  - pot
+DocGrammarBear:
+  - c
+  - cpp
+  - cs
+  - default
+  - fortran
+  - golang
+  - java
+  - objective-c
+  - php
+  - python
+  - python3
+  - tcl
+  - vhdl
+DockerfileLintBear:
+  - Dockerfile
+DocumentationStyleBear:
+  - c
+  - cpp
+  - cs
+  - default
+  - fortran
+  - golang
+  - java
+  - objective-c
+  - php
+  - python
+  - python3
+  - tcl
+  - vhdl
+DuplicateFileBear:
+  - All
+ESLintBear:
+  - HTML
+  - JSX
+  - JavaScript
+  - Markdown
+  - Typescript
+ElmLintBear:
+  - Elm
+FilenameBear:
+  - All
+FormatRBear:
+  - R
+GNUIndentBear:
+  - C
+  - C++
+GhcModBear:
+  - Haskell
+GherkinLintBear:
+  - Gherkin
+GitCommitBear:
+  - Git
+GitCommitMetadataBear:
+  - Git
+GitLinearCommitBear:
+  - Git
+GitRevertInspectBear:
+  - Git
+GoErrCheckBear:
+  - Go
+GoImportsBear:
+  - Go
+GoLintBear:
+  - Go
+GoReturnsBear:
+  - Go
+GoTypeBear:
+  - Go
+GoVetBear:
+  - Go
+GofmtBear:
+  - Go
+HAMLLintBear:
+  - Haml
+HTMLHintBear:
+  - HTML
+HTMLLintBear:
+  - HTML
+  - Jinja2
+  - PHP
+HTTPSBear:
+  - All
+HTTPoliceLintBear:
+  - HAR
+HappinessLintBear:
+  - JavaScript
+HaskellLintBear:
+  - Haskell
+HgCommitBear:
+  - Hg
+IndentationBear:
+InferBear:
+  - Java
+InvalidLinkBear:
+  - All
+JSComplexityBear:
+  - JavaScript
+JSHintBear:
+  - JavaScript
+JSONFormatBear:
+  - JSON
+JSStandardBear:
+  - JSX
+  - JavaScript
+JavaPMDBear:
+  - Java
+Jinja2Bear:
+  - Jinja2
+JuliaLintBear:
+  - Julia
+KeywordBear:
+  - All
```

Comment:
'All' isnt a language.  It should be omitted from the yaml file.  This should be a special case.

maybe write these as

```yaml
KeywordBear: true
```

to indicate the bear exists, but has no defined languages.
`KeywordBear: false` is also a possible way of describing these bears.